benchmarks/configs/agent_single_50mb_genlog_500k_line_file.json

```diff
@@ -0,0 +1,18 @@
+{
+  // We want to consume whole file from the beginning so we set max offset to 100 MB
+  max_log_offset_size: 104857600,
+
+  // 600 KB
+  max_line_size: 614400,
```

Comment:
I can also change this to something a bit closer to 500k (e.g. 510k) to see if the memory usage drops a bit (it probably should).